deps/v8/src/arm/cpu-arm.cc

```diff
@@ -64,7 +64,7 @@ void CPU::FlushICache(void* start, size_t size) {
   // None of this code ends up in the snapshot so there are no issues
   // around whether or not to generate the code when building snapshots.
   Simulator::FlushICache(Isolate::Current()->simulator_i_cache(), start, size);
-#else
+#elif defined(__linux__)
```

Comment:
Would using `#elif !defined(__FreeBSD__)` be more appropriate here? If I understand correctly, before this change this code is compiled for all platforms where `deps/v8/src/arm/cpu-arm.cc` is compiled, and thus not necessarily only on Linux.